src/components/dashboard/SendLinkSummary.js

```diff
@@ -48,6 +48,10 @@ const SendLinkSummary = ({ screenProps, styles }: AmountProps) => {
   const [link, setLink] = useState('')
   const [loading, setLoading] = useState('')
 
+  const faceRecognition = useCallback(() => {
```

Comment:
we can remove face recognition requirement from Send, it is now only required for Claim